packages/salesforcedx-vscode-core/package.nls.json

```diff
@@ -55,5 +55,6 @@
   "force_source_deploy_in_manifest_text": "SFDX: Deploy Source in Manifest to Org",
   "force_source_delete_text": "SFDX: Delete from Project and Org",
   "force_source_delete_this_source_text": "SFDX: Delete This from Project and Org",
-  "force_config_set_org_text": "SFDX: Set a Default Org"
+  "force_config_set_org_text": "SFDX: Set a Default Org",
+  "internal_dev_description": "Specifies whether or not to set up internal development mode which provides limited SFDX commands."
```

Comment:
@ruthemmanuelle this is the description users will get when they set the `internal_development` preference.

Reply:
Hi @lcampos . Does the pref do anything in addition to limiting the commands that are available?

Reply:
@ruthemmanuelle it's also meant for developing only lightning and/or lwc